_topics/en/vaults.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Vaults
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Scripts and Addresses
+  - Security Enhancements
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Vaults** are a type of covenant that requires two separate
+  transactions to appear in two different blocks in order for a user to
+  spend money from their wallet.  The first transaction signals that
+  someone is attempting to spend the money and gives the user a chance
+  to block the second transaction that completes the spend.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  Usually, there's a minimum amount of time or number of blocks that
+  must pass between the two transactions, giving the user that amount of
+  time to notice if someone stole their private key and is attempting to
+  steal their money.  If the user detects the theft attempt, most vault
+  designs also allow the user to either send the money to a safe address
+  that uses a more secure script or to permanently destroy the money
+  to prevent the thief from profiting from their attack.
+
+  Some vault designs rely on [covenants][topic covenants] that require
+  consensus changes to Bitcoin.  Other vault designs use existing
+  protocol features plus techniques such as multisig, presigned
+  transactions, and ephemeral private keys.
```

Comment:
I don't think 'ephemeral' emphasizes the right thing. The critical part is that the keys are **deleted**.